invenio_vocabularies/assets/semantic-ui/js/invenio_vocabularies/src/contrib/forms/Funding/CustomAwardForm.js

```diff
@@ -30,6 +30,8 @@ function CustomAwardForm({ deserializeFunder, selectedFunding }) {
 
     if (funderItem.country_name) {
       funderCountry = funderItem.country_name;
+    } else if (funderItem.country) {
+      funderCountry = funderItem.country;
     }
```

Comment:
I think this could be a one liner:

```javascript
const funderCountry =  funderItem ?. country_name  ?? funderItem ?. country
```

You can check running this code (e.g. in a js fiddle):

```javascript
let funderItem = {}
let funderCountry =  funderItem ?. country_name  ?? funderItem ?. country
console.log(funderCountry)
// Expected: undefined 

funderItem['country_name'] = 'Portugal'
funderCountry =  funderItem ?. country_name  ?? funderItem ?. country
console.log(funderCountry)
// Expected: 'Portugal'

delete funderItem['country_name']
funderItem['country'] = 'PT'
funderCountry =  funderItem ?. country_name  ?? funderItem ?. country
console.log(funderCountry)
// Expected: 'PT'
```

Reply:
Done, I also simplified the other fields. Optional chaining returns `undefined` while before we were defaulting to `null`, but it does not make a difference since the rest of the function is relying on truthy/falsy and both `undefined` and `null` behave the same way in this regard.